server/models/chat.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../config/db');
+
+exports.create = ({ body, id_to, id_from }) => db.one(
+  `INSERT INTO message (body, sender_id, recipient_id)
+      VALUES ($1, $2, $3)
+      RETURNING message_id`,
+  [body, id_to, id_from]
+);
+
+exports.read = (account_id) => db.any(
+  `SELECT *
+      FROM message
+      WHERE sender_id = $1 OR recipient_id = $1`,
+  [account_id]
+);
+
+exports.update = ({ message_id, body }) => db.none(
```

Comment:
Maybe this model function isn't needed, do we want to give people the option to edit messages? (I am of the opinion that this might complicate our frontend too much). I'd like to hear your opinions @jason-crowley @AdamWang00

Reply:
Yeah I'm fine leaving it out, especially because we can always add the functionality back later if we find a need for it.